nuclear/module/python/nuclear.py

```diff
@@ -152,11 +158,29 @@ def include_paths(self):
     def function(self):
         return self.func
 
+
+def is_sequence(item):
+    if not hasattr(item, "strip") and hasattr(item, "__getitem__") or hasattr(item, "__iter__"):
+        return True
+    else:
+        return False
+
+
+def bind_emit(self, msg):
+    # Iterable type
+    if is_sequence(msg):
+        return msg[0]._emit_vector(self._reactor_ptr, msg)
```

Comment:
Still don't know how I feel about this emit vector thing, We're trying to get rid of it in the rest of the code too

Reply:
Well until it is completely removed from the rest of code this needs to stay